project/ProjectPlugin.scala

```diff
@@ -114,6 +120,24 @@ object ProjectPlugin extends AutoPlugin {
       )
     )
 
+    lazy val exampleSeedProtobufProtocolSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies ++= Seq(
+         mu("mu-rpc-fs2"), 
+         mu("mu-rpc-service")
+      ),
+      muSrcGenIdlType := IdlType.Proto,
+      muSrcGenIdiomaticEndpoints := true
+    )
+
+    lazy val exampleSeedAvroProtocolSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies ++= Seq(
+         mu("mu-rpc-fs2"), 
```

Comment:
**question**: Are you using the fs2 dependency? Avro standard doesn't support streaming 🤔